codestodians/food-basket-front-end
# Update Apollo Client Configuration

#### What does this PR do?
- It updates the Application's apollo client configuration.

#### Description of Task to be completed?
- Update dependencies.
- Upgrade the configuration options in `App.jsx`.
- Update the way certain modules are imported by adding `index.jsx` files where necessary. 
- Ensure that the App works as before.

#### How should this be manually tested?
- Check out into the `chore-update-apollo-client-config-153111297` branch.
- Install dependencies by typing `yarn install` in the terminal.
- Open the App in the browser and check that all links work perfectly.
- Check the browser console to note that there are no errors.

#### Any background context you want to provide?
- As technology and frameworks are a dynamic duo, it becomes necessary to keep in touch with changes as they are made. On the one hand, it ensures that we keep abreast of developments, on the other hand, it ensures we don't fall into a situation of catastrophic/critical application failures in the case that frameworks become deprecated.

#### What are the relevant pivotal tracker stories?
- [153111297](https://www.pivotaltracker.com/story/show/153111297)
